geomstats/_backend/numpy/autograd.py

```diff
@@ -0,0 +1,16 @@
+def value_and_grad(objective):
+    """Return an error when using automatic differentiation with numpy."""
+    raise RuntimeError(
+        "Automatic differentiation is not supported with numpy backend. "
+        "Use autograd, pytorch or tensorflow backend instead.\n"
+        "Change backend via the command "
+        "export GEOMSTATS_BACKEND=autograd in a terminal.")
```

Comment:
We need quotes around `autograd` in the command line

Reply:
Actually it works without!